config/olm/bundle/manifests/flux-operator.clusterserviceversion.yaml

```diff
@@ -15,6 +15,7 @@ metadata:
             "namespace": "flux-system",
             "annotations": {
               "fluxcd.controlplane.io/reconcileEvery": "1h",
+              "fluxcd.controlplane.io/reconcileArtifactEvery": "10m",
```

Comment:
We don't set the artifact in the OLM example so this annotation can be removed.

Reply:
fixed, thanks!